src/components/previewers/ImagePreviewer.jsx

```
@@ -49,108 +62,142 @@ const ImagePreviewer = ({
+                  className="w-5 h-5 cursor-pointer text-gray-700 hover:text-blue-500"
+                  onClick={handleDownload}
+                />
+              </div>
+              <div className="w-full h-[1px] bg-gray-200 my-2"></div>
+              <div className="action-buttons-container">
+                <div className="flex items-center gap-4">
+                  <button
+                    onClick={handleLike}
+                    className="action-button"
+                    aria-label="Like"
+                  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)